specifications.yml

```diff
@@ -948,6 +1023,33 @@ components:
             type: string
             description: Description
             example: Automatic grant
+    
+    BodyForCustomerApplication:
+      type: object
+      required:
+        - decision
+      properties:
+        decision:
+          $ref: "#/components/schemas/manualDecision"
+        decisionReason:
+          type: string
+          description: Reason to justify the decision taken. (Look at the examples on the right to understand when it's required)
+        terms:
+          $ref: "#/components/schemas/termsForCustomerApplication"
+
+    BodyForFinancingApplication:
+      type: object
+      required:
+        - decision
+      properties:
+        decision:
+          $ref: "#/components/schemas/manualDecision"
+        decisionReason:
+          type: string
+          description: Reason to justify the decision taken. (Look at the examples on the right to understand when it's required)
```

Comment:
Same here, I also don't know if we should use "justify". The reason does not really need to be justified :smile: Its enough to say `Reason for the decision taken`